python/mlcroissant/mlcroissant/_src/structure_graph/nodes/metadata.py

```diff
@@ -343,12 +329,12 @@ class Metadata(Node):
         input_types=[SDO.Text],
         url=constants.ML_COMMONS_RAI_DATA_BIASES,
     )
-    data_usecases: str | None = mlc_dataclasses.jsonld_field(
+    data_use_cases: str | None = mlc_dataclasses.jsonld_field(
         default=None,
         input_types=[SDO.Text],
         url=constants.ML_COMMONS_RAI_DATA_USE_CASES,
     )
-    data_limitation: str | None = mlc_dataclasses.jsonld_field(
+    data_limitations: str | None = mlc_dataclasses.jsonld_field(
```

Comment:
Can you adapt the right cardinality: `cardinality="MANY"`

Reply:
Added cardinality “many” and “one” concerning the 1.0 SPEC to any field. 

This new structure is much clearer when it comes to performing changes, congrats :)!!!.